### ISSUE TITLE
Control font size of code

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#772]
### What are you trying to do that you can't?

Some creators like to make code smaller or bigger, depending on their access needs.

### What is your idea?

Some youth have asked for the ability to increase or decrease the size of code, without having to change the font size of the rest of the interface. We could add increase/decrease buttons to the editor toolbar, and persist the preferred size locally on the device. There are several design choices around what the min and max size should be, whether the controls should be visible or hidden by default in the toolbar.

**Comment by ksiyaaa** [comment]
@amyjko May I post some of my ideas about this issue?

**Comment by amyjko** [comment]
Please do, issues are where we discuss ideas. Are you also asking to be assigned to this issue?

**Comment by ksiyaaa** [comment]
Yes, I also wanna try this issue. @amyjko 
Here are some of my ideas towards this:

UI Design

1. Add [A-] [16px] [A+] buttons to the editor toolbar (right side, near settings)
2. Show current font size in the middle
3. Buttons should be visible by default to help students discover the feature

Settings

1. Size range: 12px minimum, 28px maximum
2. Default size: 16px
3. Increment: 2px or just 1px per click (For choosing 1px or 2px, I think we can actually try it to see how the overall interface looks like when implementing this function)
4. Store preference in localStorage

Controls

1. Click [A+] to increase by 2px/ 1px
2. Click [A-] to decrease by 2px/ 1px
3. We can also add ideal Keyboard shortcuts: Ctrl/Cmd +/- to adjust, Ctrl/Cmd 0 to reset
4. Disable buttons when reaching min/max

Additional Thoughts

I think we could also add "hide font controls" option in Settings later if toolbar gets crowded.

**Comment by amyjko** [comment]
Okay, I assigned you. A few bits of design feedback:

* We want to avoid biasing the platform to a particular language, so "A-" and "A+" aren't great labels, because they privilege latin characters. Can you think of a label that doesn't privilege a language?
* What font size unit would this design display?
* Would the preference be per source file, per project, per device, or per account? (e.g., when there are multiple source files, can they have different sizes?)
* The current toolbar ribbon distinguishes between "important" controls and other controls. Other controls are hidden in the ribbon. We'd have to decide if this is important enough for it to be visible to all learners, or just those who want this feature and are willing to dig into the ribbon for it.

**Comment by amyjko** [comment]
@ksiyaaa Are you still working on this?
